sos/report/plugins/openstack_nova.py

```diff
@@ -215,15 +216,46 @@ def setup(self):
             "/etc/polkit-1/localauthority/50-local.d/50-nova.pkla",
             "/etc/sudoers.d/nova",
             "/etc/security/limits.d/91-nova.conf",
-            "/etc/sysconfig/openstack-nova-novncproxy"
+            "/etc/sysconfig/openstack-nova-novncproxy",
+            "/var/lib/openstack/config/nova",
+            "/var/lib/openstack/containers/nova*.json"
         ])
+
         if self.get_option("all_logs"):
             self.add_copy_spec([
                 "/var/log/httpd/placement*",
+                "/var/log/containers/nova/*"
             ])
         else:
             self.add_copy_spec([
                 "/var/log/httpd/placement*.log",
+                "/var/log/containers/nova/*.log"
             ])
 
+        self.add_forbidden_path([
+            "/var/lib/openstack/config/nova/ssh-privatekey"
+        ])
+
+    def postproc(self):
+        protect_keys = [
+            "ldap_dns_password", "neutron_admin_password",
+            "rabbit_password", "qpid_password", "powervm_mgr_passwd",
+            "virtual_power_host_pass", "xenapi_connection_password",
+            "password", "host_password", "vnc_password",
+            "admin_password", "connection_password",
+            "memcache_secret_key", "s3_secret_key",
+            "metadata_proxy_shared_secret", "fixed_key", "transport_url",
+            "rbd_secret_uuid",
```

Comment:
could we just not add `rbd_secret_uuid` to the main `postproc()` definition? that way it would work for both Ubuntu and RedHat? Otherwise the whole `protect_keys` list looks identical

Reply:
sure! that makes a lot of sense.

Thanks @arif-ali